### PR TITLE
Messaging 171

### PR DESCRIPTION
**Pull request by pjaudiomv**
#171

### CODE REVIEW

**Comment by pjaudiomv** [comment]
had issues merging in recent changes so just killed branch and brought back

**Comment by radius314** [comment]
This could be further simplified by having the function return true if the setting is false. That would allow you to remove the “else if” condition.

**Comment by pjaudiomv** [comment]
im so silly

**Comment by radius314** [comment]
You could just default is_mobile to true and not check again for the setting.

**Comment by radius314** [comment]
Can you remove the debug statement?

File: endpoints/meeting-search.php [hunk 1]
```diff
@@ -26,9 +26,21 @@
     $text_space = " ";
     $message = "";
 
+function mobileCheck()
+{
+    $is_mobile = true;
+    if (has_setting('mobile_check') && json_decode(setting('mobile_check'))) {
+        $phone_number = $GLOBALS['twilioClient']->lookups->v1->phoneNumbers($_REQUEST['From'])->fetch(array("type" => "carrier"));
+        if ($phone_number->carrier['type'] !== 'mobile') {
+            $is_mobile = false;
```
Comment (jbraswell) [inline endpoints/meeting-search.php]: I would just `return false;` here, and then `return true;` at the bottom of the function. That way you don't even need `$is_mobile`".

It's already merged, and the code works, so no need to change it - but next time :).

Comment (pjaudiomv) [reply endpoints/meeting-search.php]: ah yes that totes makes sense. I initially was using $is_mobile but after code changes its no longer needed.

**Pull request merged**